### PR TITLE
Local users policy merge

### PR DESCRIPTION
**Pull request by joroca**

### CODE REVIEW

File: gecoscc/commands/check_local_users_policy.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)    
+    
+class Command(BaseCommand):
+    description = """
+       Check Users policy for all the workstations in the database. 
+       This script must be executed on major policies updates when the changes in the policies structure may
+       cause problems if the Chef nodes aren't properly updated.
+       
+       So, the curse of action is:
```
Comment (alfem) [inline gecoscc/commands/check_local_users_policy.py]: "curse"?

File: gecoscc/tasks.py [hunk 2]
```diff
@@ -494,19 +494,21 @@ def group_by_multiple_keys(self, input_data, mergeIdField, mergeActionField, opp
         from itertools import groupby
         from operator import itemgetter
 
+        self.log("debug","tasks.py ::: group_by_multiple_key - input_data = {0}".format(input_data))
         grouper = itemgetter(*mergeIdField)
```
Comment (alfem) [inline gecoscc/tasks.py]: Please, use clear, significant names for variables. A "grouper" is a fish.

File: gecoscc/commands/check_local_users_policy.py (hunk 1 above)
Comment (joroca) [inline gecoscc/commands/check_local_users_policy.py]: Fixed.

File: gecoscc/tasks.py (hunk 2 above)
Comment (joroca) [inline gecoscc/tasks.py]: Fixed.

**Comment by joroca** [comment]
I've already resolved the conflicts

**Pull request merged**